example/docker/docker-compose.yml

```diff
@@ -34,6 +34,7 @@ services:
     ports:
       - "15671:5671"
       - "15672:5672"
+      - "15673:5673"
```

Comment:
this is the internal port used by Hono server. No need to expose it to the host system. Please remove.